lib/supply/uploader.rb

```diff
@@ -13,13 +13,21 @@ def perform_upload
 
           listing = client.listing_for_language(language)
 
-          upload_metadata(language, listing)
-          upload_images(language)
-          upload_screenshots(language)
+		  if !skip_upload_metadata
+			  upload_metadata(language, listing)
+		  end
+		  if !skip_upload_images
+			  upload_images(language)
+		  end
+		  if !skip_upload_screenshots
```

Comment:
Inconsistent indentation detected.<br>Favor `unless` over `if` for negative conditions.<br>Tab detected.